src/main/java/org/moltimate/moltimatebackend/controller/ControllerExceptionHandler.java

```diff
@@ -38,4 +38,9 @@
     protected ResponseEntity<Object> invalidFile(InvalidFileException ex, WebRequest request) {
         return handleExceptionInternal(ex, ex.getMessage(), new HttpHeaders(), HttpStatus.BAD_REQUEST, request);
     }
+
+    @ExceptionHandler(value = {MotifFileParseException.class})
```

Comment:
This is only used in `FileUtils : readMotifFile`when it is not a `InvalidFileException`.

This gives us a generic runtime response message for having difficulty parsing a file.